Tests/fixtures/tv-1.yml

```diff
@@ -4092,23 +4092,1200 @@
   browser_family: Chrome
 -
-  user_agent: Mozilla/5.0 (Linux; arm; Android 9; 32LH7030S) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.85 YaApp_Android/21.111.1/apad YaSearchBrowser/21.111.1/apad BroPP/1.0 SA/3 Mobile Safari/537.36
+  user_agent: Opera/9.80 (Linux mips; Opera TV/39) Presto/2.11.355 Version/12.11
   os:
-    name: Android
-    version: "9"
-    platform: ARM
+    name: Opera TV
+    version: ""
+    platform: MIPS
   client:
     type: browser
-    name: Yandex Browser
-    version: 21.111.1
-    engine: Blink
-    engine_version: ""
+    name: Opera
+    version: "12.11"
+    engine: Presto
+    engine_version: 2.11.355
   device:
     type: tv
-    brand: Asano
-    model: 32LH7030S 32.0"
-  os_family: Android
-  browser_family: Unknown
+    brand: ""
+    model: ""
+  os_family: GNU/Linux
+  browser_family: Opera
+-
+  user_agent: BrightSign/6.0.13 (XD1132) Mozilla/5.0 (compatible; Linux mips) AppleWebKit/537.4 (KHTML, like Gecko) Chromium/18.0.1025.168 Chrome/18.0.1025.168 Safari/537.4
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: MIPS
+  client:
+    type: browser
+    name: Chromium
+    version: 18.0.1025.168
+    engine: WebKit
+    engine_version: "537.4"
+  device:
+    type: tv
+    brand: BrightSign
+    model: XD1132
+  os_family: GNU/Linux
+  browser_family: Chrome
+-
+  user_agent: Opera/9.80 (Linux mips) Presto/2.12.407 Version/12.51 , TiVo-Device/b-tcdclient-harmony/2013.12.17-2041 (TiVo, TCD8F6000, wired)
+  os:
+    name: Opera TV
+    version: ""
+    platform: MIPS
+  client:
+    type: browser
+    name: Opera
+    version: "12.51"
+    engine: Presto
+    engine_version: 2.12.407
+  device:
+    type: tv
+    brand: TiVo
+    model: TCD8F6000
+  os_family: GNU/Linux
+  browser_family: Opera
+-
+  user_agent: Opera/9.80 (Linux mips) Presto/2.12.407 Version/12.50 , Vizio Co-Star LT/1.0.3T4 (Vizio, ISV-B11, wireless)
+  os:
+    name: Opera TV
+    version: ""
+    platform: MIPS
+  client:
+    type: browser
+    name: Opera
+    version: "12.50"
+    engine: Presto
+    engine_version: 2.12.407
+  device:
+    type: tv
+    brand: Vizio
+    model: ISV-B11
+  os_family: GNU/Linux
+  browser_family: Opera
+-
+  user_agent: Opera/9.80 (Linux mips; ) Presto/2.12.407 Version/12.51 MB95/3.3.6.X (TOSHIBA, Si2156LG32, wireless)
+  os:
+    name: Opera TV
+    version: ""
+    platform: MIPS
+  client:
+    type: browser
+    name: Opera
+    version: "12.51"
+    engine: Presto
+    engine_version: 2.12.407
+  device:
+    type: tv
+    brand: Toshiba
+    model: Si2156LG32
+  os_family: GNU/Linux
+  browser_family: Opera
+-
+  user_agent: Opera/9.80 (Linux mips; ) Presto/2.12.407 Version/12.51 MB95/3.6.3.J (POLAROID, Si2157LG32, wireless)
+  os:
+    name: Opera TV
+    version: ""
+    platform: MIPS
+  client:
+    type: browser
+    name: Opera
+    version: "12.51"
+    engine: Presto
+    engine_version: 2.12.407
+  device:
+    type: tv
+    brand: Polaroid
+    model: Si2157LG32
```

Comment:
model: ""

`Si2157LG32`, `Mxl661L32`, `` is a panel/mainboard  models that is installed on several brands
JVC, LG, MEDION, Polaroid, HITACHI, LUXOR etc